fdbclient/include/fdbclient/MultiVersionTransaction.h

```diff
@@ -1104,6 +1104,7 @@ class MultiVersionApi : public IClientApi {
 
 	bool networkStartSetup;
 	volatile bool networkSetup;
+	volatile bool disableBypass;
```

Comment:
We probably shouldn't actually be using `volatile` on any of these, and we can omit it for new ones. We should clean up the existing ones at some point too, but that doesn't need to be here.

```suggestion
	bool disableBypass;
```